l3packages/xtemplate/xtemplate.dtx

```diff
@@ -707,2225 +690,47 @@
 %    \end{macrocode}
 %
 %    \begin{macrocode}
-%<@@=xtemplate>
+%<@@=template>
```

Comment:
A latex3 prefix renaming. Corresponding entries in [`l3prefixes.csv`](https://github.com/latex3/latex3/blob/main/l3kernel/doc/l3prefixes.csv) need an update?

Reply:
We should not free xtemplate in my opinion but reserve both.

Reply:
We never release prefixes once taken, as older code will still use them. As the change is for compatibility, I don't think we need an update: someone finding `xtemplate` in an internal macro is still best pointed to the L3 repo, and someone finding `template` to the 2e one.